.github/workflows/publish_package_testpypi.yaml

```
@@ -0,0 +1,40 @@
+name: Publish to Test PyPI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upload_to_pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # This is required to properly tag packages
+
+      - name: Use Node.js 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Install node modules
+        run: |
+          cd clients/admin-ui
+          npm install
+
+      - name: Build and export frontend files
+        run: |
+          cd clients/admin-ui
+          npm run prod-export
+
+      - name: Install Twine
+        run: pip install twine
+
+      - name: Twine Upload
+        run: |
+          python setup.py sdist bdist_wheel 
+          twine upload --repository testpypi dist/*
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_TOKEN }}
```

I know this is the way we are doing it in all our repos and it is working but if seems fragile. I don't think Ubuntu has `pip` at all and if it does it would be Python 2. Github must be installing it in their image and maybe sym linking it to `pip3`.

The way it is currently has been working fine all along so this isn't a blocker, just something I noticed.

```suggestion
      - name: Setup Python 3.8
        uses: actions/setup-python@v2
        with:
          python-version: 3.8
          
      - name: Install Twine
        run: pip install twine
        
      - name: Twine Upload
        run: |
          python setup.py sdist bdist_wheel 
          twine upload --repository testpypi dist/*
        env:
          TWINE_USERNAME: __token__
          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_TOKEN }}
```